### PR TITLE
Tag OnlineStats v0.1.1

### PR DESCRIPTION
**Pull request by joshday**
Remove Convex and SCS dependencies

### CODE REVIEW

File: OnlineStats/versions/0.1.1/requires [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Docile
+Compat
+Reexport
+FactCheck
```
Comment (IainNZ) [inline OnlineStats/versions/0.1.1/requires]: This should probably be in `test/REQUIRE`

File: OnlineStats/versions/0.1.1/requires [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+StatsBase
+Distributions
+
+MultivariateStats
+GLM
+Dates
+Reactive
```
Comment (IainNZ) [inline OnlineStats/versions/0.1.1/requires]: This seems commented out in your code?

File: OnlineStats/versions/0.1.1/requires [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Docile
```
Comment (IainNZ) [inline OnlineStats/versions/0.1.1/requires]: Should have `julia 0.3` as first line

File: OnlineStats/versions/0.1.1/requires (hunk 2 above)
Comment (joshday) [inline OnlineStats/versions/0.1.1/requires]: What seems commented out?

File: OnlineStats/versions/0.1.1/requires (hunk 1 above)
Comment (joshday) [inline OnlineStats/versions/0.1.1/requires]: Will do.

File: OnlineStats/versions/0.1.1/requires (hunk 3 above)
Comment (joshday) [inline OnlineStats/versions/0.1.1/requires]: Will do.

File: OnlineStats/versions/0.1.1/requires (hunk 2 above)
Comment (IainNZ) [inline OnlineStats/versions/0.1.1/requires]: Reactive:
https://github.com/joshday/OnlineStats.jl/blob/42e4119366b0f85ad54e2c349ec1dca10376e2e0/src/OnlineStats.jl#L31

Comment (joshday) [inline OnlineStats/versions/0.1.1/requires]: Ah, my collaborator's work...I'll look into that.

**Comment by joshday** [comment]
Sorry for cluttering with pull requests.  See #3141.

**Pull request closed without merging**